cluster_test.go

```diff
@@ -1682,7 +1682,7 @@ func (s *S) TestMaxSocketUses(c *C) {
 	if *fast {
 		c.Skip("-fast")
 	}
-	session, err := mgo.Dial("localhost:40011?maxSocketUses=4")
```

Comment:
I see some removals later on, but I think the variable is still in the struct. Are we trying to replace this with the connection life time, or support both?